python/lsst/alert/packet/schema.py

```diff
@@ -174,6 +174,10 @@ def __init__(self, schema_definition):
         self.definition = resolve_schema_definition(schema_definition)
         fastavro.schema._schema.SCHEMA_DEFS.clear()
 
+
+    def __eq__(self, other):
+        return self.definition == other.definition
```

Comment:
This looks pretty similar to https://github.com/lsst/alert_packet/blob/c3615d8b0b284fea833756849eb484ea1aa421f0/python/lsst/alert/packet/schema.py#L276...?

Reply:
It sure does, doesn't it. I don't know why a particular test had been failing intermittently, then, but it doesn't seem to fail anymore.

Reply:
Incidentally, today I learned that you can declare a method twice inside one class. Why on earth is that allowed? Ah, Python.